TicTacToe2/TicTacToe/Board.cs

```diff
@@ -6,5 +6,74 @@ namespace TicTacToe2.TicTacToe
 {
     class Board
     {
+        private char[,] space; //Board
+        private int gridNumber = 0; //Board
+
+        private void ChooseBoard() //Board
+        {
+            Console.WriteLine("\nThis grid is user defined.");
+            Console.WriteLine("ie.Inputing 3 will result in a grid 3 rows by 3 columns.");
+            gridNumber = GetUserInput(3, 1000);
+            space = new char[gridNumber, gridNumber];
+        }
+
+        private void InitializeBoard() //Board
+        {
+            for (int row = 0; row < gridNumber; row++)
+            {
+                for (int col = 0; col < gridNumber; col++)
+                {
+                    space[row, col] = ' ';
+                }
+            }
+        }
+
+        private void DrawBoard()  //Board
```

Comment:
This will need to be called by the game, so it should be part of the class's interface (public, not private)